dev-router.php

```diff
@@ -0,0 +1,18 @@
+<?php
+// This file is used for testing BzDeck with PHP's built-in web server.
+// Usage: 
+// 1. php -S localhost:8000 dev-router.php
+// 2. Visit http://localhost:8000
+
+if (file_exists(__DIR__ . '/webroot' . $_SERVER['REQUEST_URI'])) {
+  // Serve the requested resource as-is.
+  return false;
+}
+
+if ($_SERVER['REQUEST_URI'] === '/static/scripts/combined.js') {
+  // Map combined.js to PHP
+  include('components/combine-scripts.php');
```

Comment:
It worked for me this morning, I guess the difference is the `-t webroot` parameter I used in `run_dev_server.sh`. I'll double check when I get to office tomorrow.